test/built-ins/BigInt/prototype/Symbol.toStringTag.js

```diff
@@ -19,7 +19,7 @@ verifyProperty(BigInt.prototype, Symbol.toStringTag, {
   writable: false,
   enumerable: false,
   configurable: true
-});
+}, {restore: true});
```

Comment:
restore is set here for a bad practice for tests. Tests should be atomic and assertions should not rely on other assertion behaviors or even order of execution in the matters of any other assertions.

The best practice here would be isolating the assertions if they can break or explode others. In Test262 case, we should split this into multiple files. It's fun that `restore` would become totally useless after that, because it does affect the assertion it's applied to.

Reply:
This should be applied to all cases relying on this restore, I don't see a good reason to use it otherwise.

Reply:
> This should be applied to all cases relying on this restore, I don't see a good reason to use it otherwise

Agreed.

Reply:
> restore is set here for a bad practice for tests. Tests should be atomic and assertions should not rely on other assertion behaviors or even order of execution in the matters of any other assertions.

> The best practice here would be isolating the assertions if they can break or explode others. In Test262 case, we should split this into multiple files. It's fun that restore would become totally useless after that, because it does affect the assertion it's applied to.

Agreed. I will break these out.